packages/dashboard/app/components/DirectoryPicker.tsx

```
@@ -100,7 +105,63 @@ export function DirectoryPicker({ value, onChange, placeholder, onInputKeyDown,
     if (browser.isOpen && browser.currentPath) {
       fetchEntries(browser.currentPath, browser.showHidden);
     }
-  }, [browser.showHidden]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)